src/core/elements/getElements.js

```diff
@@ -27,16 +28,20 @@ const downloadElements = async (elements, query, jobId, processId, isPrivate) =>
         });
         return null;
       }
+
       logDebug(`Downloading element for element key - ${element.key}`);
       const exportedElement = await http.get(makePath(element), query);
       logDebug(`Downloaded element for element key - ${element.key}`);
+
       emitter.emit(EventTopic.ASSET_STATUS, {
         processId,
         assetType: Assets.ELEMENTS,
         assetName: element.key,
-        assetStatus: ArtifactStatus.COMPLETED,
+        assetStatus: equals(jobType, JobType.PROMOTE_EXPORT) ? ArtifactStatus.INPROGRESS : ArtifactStatus.COMPLETED,
```

Comment:
@mukundbayyaram You have to do the same for formulas and VDRs as well.

Reply:
Done